apps/andi/test/unit/andi/secret_service_test.exs

```diff
@@ -25,14 +25,26 @@ defmodule Andi.SecretRetrieverTest do
       allow Vault.read(values.vault, "secrets/smart_city/auth0/andi"),
         return: {:ok, values.credentials}
 
-      assert SecretRetriever.retrieve_auth0_credentials() == {:ok, values.credentials}
+      assert SecretService.retrieve_auth0_credentials() == {:ok, values.credentials}
```

Comment:
How are we handling failures for this? Do we expect them to crash the page? Or are we dealing with them with an error tuple?

Reply:
As in, errors other than the `read` ones we've dealt with

Reply:
Right now they are crashing the app

Reply:
Oh good